core/ibft.go

```diff
@@ -453,7 +452,7 @@ func (i *IBFT) handleRoundChangeMessage(view *proto.View, quorum uint64) *proto.
 		isValidFn,
 	)
 
-	if len(msgs) < int(quorum) {
+	if i.backend.HasQuorum(view, msgs) {
```

Comment:
I believe we need negation here:

```suggestion
	if !i.backend.HasQuorum(view, msgs) {
```